src/components/UI/Avatar.vue

```diff
@@ -42,7 +50,7 @@ const classes = getStyles(props, 'avatarStyles')
             v-if="avatarImg"
             :src="avatarImg"
             :alt="avatarAlt"
-            class="w-full h-full rounded-full"
+            :class="['w-full', 'h-full', 'rounded-full', avatarModal.showing ? 'rounded-none' : '',]"
```

Comment:
You could maybe pass this as a template string `w-full h-full ${avatarModal.showing ? 'rounded-none' : ''}`  This works, but its the only place we are passing classes as an array like this.